db/migrate/20160427110629_create_service_interactions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateServiceInteractions < ActiveRecord::Migration
+  def change
+    create_table :service_interactions do |t|
+      t.integer :service_identifier
+      t.string :service_label
```

Comment:
I would expect that we could have only 2 columns on this table, `service_id` and `interaction_id`, referring to the `id` columns of services and interactions, like foreign keys. I don't think we should duplicate the `label` or `identifier` fields in this table - it would be more work to keep them up to date when they change in future.